Azure/sonic-buildimage
# Fix bgpmon.py packaging issue for console_scripts entry point

This is to fix "https://github.com/Azure/sonic-buildimage/pull/5329/files" where the bgpmon code itself was not properly packaged for the console_scripts entry point and causing the bgpmon daemon unable to start since bgpmon.py  was not found.
it was not caught earlier due to the build was not cleaned up properly that installed the bgpmon.py via the scripts method.

Signed-off-by: Gen-Hwa Chiang <gechiang@microsoft.com>

**- What I did**

Created a new project directory and moved bgpmon.py into that directory for packaging purpose.
Change the setup.py file to indicate where to expect the bgpmon.py in the console_scripts specification.

**- How to verify it**

once the BGP docker is up and running, you can perform "show services" to see that this new daemon "bgpmon.py" is running under the BGP docker service similar to the following output:
```
...
bgp     docker
---------------------------
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.4  0.2  28328 22660 pts/0    Ss+  23:16   0:03 /usr/bin/python2 /usr/bin/supervisord
root        21  0.0  0.2  22664 16708 pts/0    S    23:16   0:00 python /usr/bin/supervisor-proc-exit-listener --container-name bgp
root        25  0.0  0.0 225856  3564 pts/0    Sl   23:16   0:00 /usr/sbin/rsyslogd -n -iNONE
frr         29  6.2  0.4 524052 35276 pts/0    Sl   23:16   0:50 /usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm -M snmp
frr         32  0.0  0.0  43308  6188 pts/0    S    23:16   0:00 /usr/lib/frr/staticd -A 127.0.0.1
frr         33  3.6  0.9 355520 80412 pts/0    Sl   23:16   0:29 /usr/lib/frr/bgpd -A 127.0.0.1 -M snmp
root        40  0.6  0.6  69192 55516 pts/0    S    23:16   0:05 /usr/bin/python /usr/local/bin/bgpcfgd
root        41  0.0  0.1  20960 15180 pts/0    S    23:16   0:00 /usr/bin/python /usr/local/bin/bgpmon
root        42  1.5  0.0  82080  6060 pts/0    Sl   23:16   0:12 fpmsyncd
...
```
